### PR TITLE
fix(RELEASE-1058): fix linting issues in cleanup-workspace task

### PR DESCRIPTION
**Pull request by jinqi7**

### CODE REVIEW

File: tasks/cleanup-workspace/README.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ Tekton task to delete a given directory in a passed workspace and cleanup Intern
 | delay          | Time in seconds to delay execution. Needed to allow other finally tasks to access workspace before being deleted | Yes      | 60            |
 | pipelineRunUid | The uid of the current pipelineRun. It is only available at the pipeline level                                   | Yes      | ""            |
 
+## Changes in 0.8.1
+* fix linting issues in this task
```
Comment (mmalina) [inline tasks/cleanup-workspace/README.md]: Capital first letter, please.

File: tasks/cleanup-workspace/cleanup-workspace.yaml [hunk 2]
```diff
@@ -45,11 +45,11 @@ spec:
...
         echo "Delaying execution by $(params.delay) seconds"
-        sleep $(params.delay)
+        sleep "$(params.delay)"
 
         CLEANUP_DIR="$(workspaces.input.path)/$(params.subdirectory)"
 
         if [ -d "${CLEANUP_DIR}" ] ; then
           # Delete a directory and its contents
-          rm -rf "${CLEANUP_DIR}"/
+          rm -rf "${CLEANUP_DIR:?}/"
```
Comment (mmalina) [inline tasks/cleanup-workspace/cleanup-workspace.yaml]: Shellcheck probably complained that if the variable is empty, you will end up deleting `/`, right? How about just removing the trailing `/`? It seems redundant.

Comment (jinqi7) [reply tasks/cleanup-workspace/cleanup-workspace.yaml]: OK. Updated. Thanks

File: tasks/cleanup-workspace/README.md (hunk 1 above)
Comment (jinqi7) [reply tasks/cleanup-workspace/README.md]: Fixed

**Review by mmalina — DISMISSED** [review]

**Review by johnbieren — DISMISSED** [review]

**Review by mmalina — APPROVED** [review]

**Pull request merged**